.travis.yml

```diff
@@ -4,45 +4,30 @@ dist: trusty
 cache:
   directories:
     - $HOME/.cache/pip
+
 python:
   - "pypy-5.4.1"
   - "3.6"
   - "3.5"
   - "3.4"
   - "2.7"
   - "nightly"
+
 env:
   global:
     - TEST='-v --durations 25'
     - PYTHONFAULTHANDLER=x
     - PYTHONWARNINGS=all
     - SKIP_LATEX_BUILD=1
-  matrix:
-    - DOCUTILS=0.13.1
-    - DOCUTILS=0.14
```

Comment:
How do you go these matrix tests?

Reply:
We get these and more via the `du*` tests in (`du11`, `du12`, `du13`, `du14`). These are run automatically via tox. If you check the Travis log, you'll see this

Reply:
Where can I see the result of them? It seems no `du*` tasks are not executed.
https://travis-ci.org/sphinx-doc/sphinx/builds/282468735?utm_source=github_status&utm_medium=notification

Reply:
Oh, damn - I was mistaken. Turns out you need to use [Advanced Configuration](https://tox-travis.readthedocs.io/en/stable/toxenv.html#advanced-configuration) to enable this. I've fixed this now.

Reply:
@tk0miya I have now fixed the `du*` issue, along with ensuring the `docs` and `mypy` targets run. You can see the results [here](https://travis-ci.org/sphinx-doc/sphinx/builds/283292020)

Reply:
Okay, I confirmed. But it seems testing time has been increased. both total time and throughput become long. Why?
Result for this: https://travis-ci.org/sphinx-doc/sphinx/builds/283292020
master branch: https://travis-ci.org/sphinx-doc/sphinx/builds/283254206

In addition, I noticed `--durations 25` option for pytest has been disabled. To know which testcases are slow, I need to enable it.